jsx/Form.js

```diff
@@ -1365,6 +1365,57 @@ StaticElement.defaultProps = {
   text: null,
 };
 
+/**
+ * Email element component.
+ * Used to displays plain/formatted email input as part of a form
+ *
+ * To pass a formatted text, you need to wrap it in a single parent element.
+ * Example usage:
+ *
+ * ```
+ * let myText = (<span>This is my <b>text</b></span>);
```

Comment:
im curious .. whats the reason behind allowing formatted text? and what is this text attribute? is it just the value of the field? i am wondering why it's not going into a 'value' attribute like the others